tests/path_util.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+
+def src_path(*args):
+    root = os.path.join(__file__, "..", "..")
+    path = os.path.join(root, *args)
+    return os.path.normpath(path)
```

Comment:
@herb I agree, but wouldn't that be equally as fragile? we would still need to do `os.path.join(__file__, "..", "..")`

Reply:
Sorry, I meant for `subprocess`'ing `grouper-fe` and `grouper-api`. But I concede the point that this doesn't apply generally to all the current use cases. We can probably get rid of the config usage by loading config instead and we could get rid of the plugin usage by cleaning up #167 and using that instead. 

But no need to burden you with that. Mind putting a todo in here to clean this up at some point?

Reply:
FWIW, the second patch we apply to the shipped config (`["common"]["plugin_dir"]`) was redundant since I updated `dev.yml`, so I removed that, meaning one fewer call site for `src_path`.

Reply:
done.